tests/scenarios/revert.spec.js

```diff
@@ -0,0 +1,142 @@
+const server = require('../server');
+const steps = require('../steps');
+
+module.exports = Object.assign(
+  {
+    before: (client, done) => {
```

Comment:
OK but we should figure out how to stop duplicating this bit.

Reply:
@boutell not sure what you are proposing here.

Reply:
The before block here looks like it's probably the same across a bunch of scenarios. Not a blocker.